talent/job-descriptions/offers/sales-sf.md

```diff
@@ -0,0 +1,36 @@
+Inside Sales Representative
+------------------------------------------
+
+### [Location & Seniority](https://github.com/src-d/guide/blob/master/talent/job-descriptions/sf-location-seniority-section.md)
+
+**Compensation:** ? [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md) for ?
+
+### [Company](https://github.com/src-d/guide/blob/master/talent/job-descriptions/company-section.md)
+
+### Role
+
+Responsibilities:
```

Comment:
@vcoisne I would suggest adding a few sentences about the team. eg. ```You will be part of the Sales team that is responsible for...``` , at least give an understanding to the candidates 1) what is the main goal of the team 2) with who is going to work.

Please feel free to add me as a reviewer in future relevanat PR's, so I won't miss them. Also, does the role need to be published on Lever or it will be kept internal for now?

Reply:
@tsolakoua right, back when I created this we did not know much about the team structure, etc.  @jorgeschnura can give more context on the position and answer the question about Lever.

Reply:
@vcoisne sounds good, thank you!